homeassistant/components/switcher_kis/config_flow.py

```diff
@@ -2,9 +2,119 @@
 
 from __future__ import annotations
 
-from homeassistant.helpers import config_entry_flow
+from collections.abc import Mapping
+import logging
+from typing import Any
 
-from .const import DOMAIN
-from .utils import async_has_devices
+from aioswitcher.bridge import SwitcherBase
+from aioswitcher.device.tools import validate_token
+import voluptuous as vol
 
-config_entry_flow.register_discovery_flow(DOMAIN, "Switcher", async_has_devices)
+from homeassistant.config_entries import ConfigEntry, ConfigFlow, ConfigFlowResult
+
+from .const import CONF_TOKEN, CONF_USERNAME, DOMAIN
```

Comment:
Use core constants:
`from homeassistant.const import CONF_USERNAME, CONF_TOKEN`

Reply:
It is done.